src/components/Button/index.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { MdKeyboardArrowRight, MdKeyboardArrowLeft } from "react-icons/md";
+
+const Button = ({
+  children,
+  type = "",
+  onClick,
+  className = "",
+  link,
+  arrowLeft,
+  arrowRight,
+}) => {
+  return link ? (
```

Comment:
Link buttons should be separated from regular buttons, split this component into to linkButton and button. this will make button usage semantically correct